pom.xml

```diff
@@ -286,13 +287,17 @@
 				<artifactId>logback-classic</artifactId>
 				<version>${version.qos.logback}</version>
 			</dependency>
-
-
 		</dependencies>
 	</dependencyManagement>
 
 	<!-- All children will have these dependencies -->
 	<dependencies>
+		<!-- jwt for auth -->
+		<dependency>
```

Comment:
this should go under dependency management since not all modules require it.

Reply:
sorry, modified